<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>예외(exception)</title>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
</head>
<body>
    <script>
        $(document).ready(async function(){
          try{
            const response = await $.ajax('http://192.168.0.34:8081/board/all')
            console.log(response);
        }catch(ex){
            if(ex.status===o){
                location.href = "error.html"
            }
            // 서버에서 논리오류가 발생하면 409 리턴되도록 만들어놓음
            if(ex.status===409){
                alert("ex.responseText");
                location.href = "/연습/list.html"
            }
        }
        })

    </script>
</body>
</html>